### ISSUE TITLE
Caffe -> IR -> TensorFlow: Unable to use converted .pb file

### ISSUE THREAD

**Issue by doudasek** [Microsoft/MMdnn#429]
**Platform** (like ubuntu 16.04/win10):
Ubuntu 16.04.5 LTS

**Python version:**
2.7.15

**Source framework** with version (like Tensorflow 1.4.1 with GPU):
Caffe 1.0.0 BVLC

**Destination framework** with version (like CNTK 2.3 with GPU):
TensorFlow 1.5.1

**Pre-trained model path** (webpath or webdisk path):
Caffe Mobilenet v1 finetuned using DIGITS 6.0.0
https://drive.google.com/open?id=12oXdKSPzmEWvXJUn3GPwsxmhcTKK7yvv

**Running scripts:**
```
mmtoir -f caffe -n deploy.prototxt -w model.caffemodel -o ir
mmtocode -f tensorflow --IRModelPath ir.pb --IRWeightPath ir.npy --dstModelPath ir.py
mmtomodel -f tensorflow -in ir.py -iw ir.npy -o tf --dump_tag SERVING
```

After running these scripts there is folder:
```
tf
├── saved_model.pb
└── variables
    ├── variables.data-00000-of-00001
    └── variables.index
```

I need frozen graph and if I want to use `saved_model.pb` for example in mobile application or use script for optimization and [quantization](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/quantization/quantize_graph.py), it throws error.
> google.protobuf.message.DecodeError: Error parsing message

**Comment by ronyuzhang** [comment]
i also face the same mistakes

**Comment by doudasek** [comment]
Finally I found solution. The error is because the graph (`tf/saved_model.pb`) is not frozen. 

I have used this [script](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tools/freeze_graph.py):
```
python ./tools/freeze_graph.py --input_saved_model_dir=./tf  --output_graph=frozen_graph.pb --output_node_names=softmax
```
But I am facing next problem because the converted model is not classifying correctly. Even if I switch RGB to BGR. Moreover one frame 224x224 is proccesed about 3-5 seconds on my Nexus 5x. So it is not usable.
